types/undertaker/index.d.ts

```diff
@@ -46,20 +52,21 @@ declare class Undertaker extends EventEmitter {
 
     /**
      * Returns the registered function.
-     * @param taskName - Task name.
      */
     task(taskName: string): Undertaker.TaskFunction;
 
     /**
      * Register the task by the taskName.
-     * @param taskName - Task name.
-     * @param fn - Task function.
      */
     task(taskName: string, fn: Undertaker.TaskFunction): void;
 
     /**
+     * Register the task by the taskName with dependencies.
+     */
+    task(taskName: string, deps: string[], fn: Undertaker.TaskFunction): void;
```

Comment:
In source file I see only two arguments:
https://github.com/gulpjs/undertaker/blob/master/lib/task.js#L3
Can you give a link to source or documentation?

Reply:
@GiedriusGrabauskas https://github.com/gulpjs/gulp/blob/de1acf66a6253302c856971f8c00f65ea6c63882/docs/API.md#gulptaskname--deps--fn